rules/macos/execution_initial_access_suspicious_browser_childproc.toml

```diff
@@ -66,6 +66,8 @@ process where host.os.type == "macos" and event.type in ("start", "process_start
       "hw.model",
       "IOPlatformExpertDevice",
       "/Volumes/Google Chrome/Google Chrome.app/Contents/Frameworks/*/Resources/install.sh",
+      "/Applications/Google Chrome.app/Contents/Frameworks/Google Chrome Framework.framework/Versions/*/Helpers/Google Chrome Helper (Renderer).app/Contents/MacOS/Google Chrome Helper (Renderer)",
```

Comment:
Might be worth checking the browser `process.parent.name` values for each browser again. Checking telem, I only see Firefox and Google Chrome, but they slightly different then how we reference them in the query. I also included Brave as it is commonly used.

<img width="1295" alt="Screenshot 2024-03-08 at 5 31 32 PM" src="https://github.com/elastic/detection-rules/assets/99630311/e52d151d-316f-4223-871a-449e624002f0">